.nycrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
c8 uses .nycrc?

Reply:
Yep
> A configuration file can be specified by passing its path on the command line with --config or -c. If no config option is provided, c8 searches for files named .c8rc, .c8rc.json, .nycrc, or .nycrc.json, starting from cwd and walking up the filesystem tree.

But you are right, we should use `.c8rc` in this case.

I'll change it tomorrow

Reply:
Moved `c8` config to `c8` section in `package.json` in favor of consistency